Vokoder.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = "Vokoder"
-  s.version          = "4.0.2"
+  s.version          = "4.1.0"
```

Comment:
I think this is a breaking change, so technically should be `5.0.0`, but I only took a quick glance, so maybe I'm wrong.